### PR TITLE
Add jjx to update Phabricator plugin

### PR DESCRIPTION
**Pull request by sectioneight**
Jerry Xie is a coworker of mine. Trying to reduce the bus factor on
releasing this plugin.

### CODE REVIEW

File: permissions/plugin-phabricator-plugin.yml [hunk 1]
```diff
@@ -4,4 +4,5 @@ paths:
 - "org/jenkins-ci/plugins/phabricator-plugin"
 developers:
 - "ai"
```
Comment (sectioneight) [inline permissions/plugin-phabricator-plugin.yml]: This is me.

**Comment by sectioneight** [comment]
https://github.com/uber/phabricator-jenkins-plugin/commit/1cfcac8827ecfd1a6e88b6047751e73c4ce7b4fb

**Comment by orrc** [comment]
LGTM.

**Comment by sectioneight** [comment]
Cool. How do we get this merged?

**Comment by oleg-nenashev** [comment]
merged. The changes should be applied within several minutes

**Comment by sectioneight** [comment]
Thanks!

**Comment by daniel-beck** [comment]
@sectioneight Please note that we discovered that a user first must have been logged in once before their account exists in Artifactory. That hasn't happened for jjx, so the permission is currently not granted.

**Comment by sectioneight** [comment]
Thanks for the note @daniel-beck.

@jjx see message above

**Comment by daniel-beck** [comment]
@sectioneight @jjx FYI my periodic check showed this is still unresolved.

**Comment by sectioneight** [comment]
Unresolved, how? @jjx, have you not been publishing this recently?

**Comment by daniel-beck** [comment]
@sectioneight 
> `"Permission target contains a reference to a non-existing user 'jjx'."`

**Comment by daniel-beck** [comment]
@sectioneight To clarify, the above is the error that Artifactory tells me. This happens when a user in LDAP has never logged in to Artifactory (UI or Maven release, doesn't matter).

**Comment by sectioneight** [comment]
@jjx please take a look

**Comment by daniel-beck** [comment]
Removing jjx to unblock https://github.com/jenkins-infra/repository-permissions-updater/pull/694 which will add automatic user list validation. File a new repo once they've logged in to add the entry again.

**Pull request merged**